ntoskrnl/kdbg/kdb_cli.c

```diff
@@ -1508,14 +1507,18 @@ KdbpCmdBreakPointList(
         else
         {
             GlobalOrLocal = Buffer;
+#define PID_NO_PROCESS ((HANDLE) -1)
             sprintf(Buffer, "  PID 0x%Ix",
-                    (ULONG_PTR)(Process ? Process->UniqueProcessId : INVALID_HANDLE_VALUE));
+                    (ULONG_PTR)(Process ? Process->UniqueProcessId : PID_NO_PROCESS));
+#undef PID_NO_PROCESS
```

Comment:
These changes are useless. Please revert.

Reply:
This is part of [CORE-10187](https://jira.reactos.org/browse/CORE-10187).
Moreover, now reduced to `DWORD` only.